src/generic/generic-component-form-label/generic-component-form-label.twig

```diff
@@ -1,21 +1,38 @@
 {#
   Parameters:
-    - "label" (string or block): label (mandatory)
-    - "for_attribute" (string): ID of the element (optional)
-    - "extra_classes" (string): extra CSS classes to be added
-    - "extra_attributes" (array): extra attributes classes (optional, format: [{ 'name': 'name_of_the_attribute', 'value': 'value_of_the_attribute'}])
+    - "label" (string) (default: '')
+    - "for_attribute" (string) (default: ''): ID of the element
+    - "extra_classes" (string) (default: ''): additional classes to add to the component
```

Comment:
nitpicking:
"extra CSS classes to be added" doesn't really add to the name of the param